jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -2264,6 +2282,184 @@ YUI.add('juju-env-go', function(Y) {
         response.changeSet = data.Response.Changes;
       }
       userCallback(response);
+    },
+
+    // EnvironmentManager facade API endpoints.
+    // XXX frankban: facade management should be improved. We should store
+    // the list of facades and their corresponding available versions upon
+    // login, and then activate/deactivate the ability to perform specific
+    // API calls based on the server capabilities. Also, facades could help
+    // namespacing client functions, so that we don't store all the API calls
+    // inside a common namespace, resulting in a more organized code. Perhaps
+    // this can be done as part of creating an external JavaScript API library.
+    environmentManagerFacadeVersion: 1,
+
+    /**
+      Create an environment within this system, using the given name.
+
+      @method createEnv
+      @param {String} envName The name of the new environment.
+      @param {String} userTag The name of the new environment owner, including
```

Comment:
To give an hint about the fact that this is not just the username (e.g. "admin") but a it is also prefixed to be a juju-core entity tag (e.g. "user-admin").